include/engine/engine.hpp

```diff
@@ -184,7 +185,7 @@ bool Engine<routing_algorithms::corech::Algorithm>::CheckCompability(const Engin
 
         auto mem = storage::makeSharedMemory(barrier.data().region);
         auto layout = reinterpret_cast<storage::DataLayout *>(mem->Ptr());
-        return layout->GetBlockSize(storage::DataLayout::CH_CORE_MARKER) > 4;
+        return layout->GetBlockSize(storage::DataLayout::CH_CORE_MARKER) > 16;
```

Comment:
This (and underlying `sizeof(std::uint64_t) + sizeof(util::FingerPrint);`) can really bite... Any suggestions for a more understandable naming, to get rid of these magic numbers?

The least we should do is add some comments to both sections.